### PR TITLE
Add impact overview (fixes #244, fixes #245, fixes #246, fixes #247, fixes #248)

### LINKED ISSUE #244: Impact form dashboard -NPS view for Rep
As a Rep I want to be able to see how I performed at my event (NPS) based on the data selected in the impact form by filtering by my name and event date.

### LINKED ISSUE #245: Impact form dashboard - type of my audience
As a Rep I want to be able to see how the type of the audience that attended my event based on the data selected in the impact form by filtering by my name and event date.

### LINKED ISSUE #246: Impact form dashboard - number of events
As a community manager I want to see the number of events that have submitted the impact form

### LINKED ISSUE #247: Impact form dashboard - Type of events
As a Reps community manager I want to see the statistics on the types of events that have submitted the impact form

### LINKED ISSUE #248: Impact form - general NPS
As the Reps community manager I want to see the general NPS from the events that have submitted the impact form

### PR DESCRIPTION
**Pull request by MichaelKohler**
**@AlexLakatos please review only, do not merge yet!**

* /impact with a full overview of all responses (grouped and anonymous)
* API to do the impact form stuff + Reps names
* Minor visual changes

screen shot 2017-07-20 at 00 57 16-fullpage

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -14,12 +14,13 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://activate.mozilla.community" # the base hostname & protocol for your site
 twitter_username: mozparticipate
 github_username:  mozilla
+api_base_url: https://activateapi.michaelkohler.info
```
Comment (MichaelKohler) [inline _config.yml]: for now running on my server, filed https://github.com/mozilla/activate.mozilla.community/issues/274 for the move to Community IT.

File: _sass/_layout.scss [hunk 2]
```diff
@@ -295,6 +299,71 @@
         .hidden {
             display: none;
         }
+
+        .impact-totals {
+            .dashboard {
+                display: -webkit-box;
```
Comment (AlexLakatos) [inline _sass/_layout.scss]: can we add a mixin to handle vendor prefixes? or maybe compass for out of the box stuff

Comment (MichaelKohler) [inline _sass/_layout.scss]: sounds good to me, will look into it

Comment (AlexLakatos) [inline _sass/_layout.scss]: do you want me to merge this and you set up a cleanup issue for across the site? 😏

Comment (MichaelKohler) [inline _sass/_layout.scss]: nope, no merge yet, thanks though :)

**Comment by MichaelKohler** [comment]
I will create a new PR with the conflict resolved. Will carry over Laka's approval.

**Pull request closed without merging**